manifests/mod/jk.pp

```diff
@@ -9,6 +9,11 @@
 # https://tomcat.apache.org/connectors-doc/reference/apache.html
 #
 class apache::mod::jk (
+  # Binding to mod_jk
+  Optional[String] $ip         = undef,
+  Pattern[/^\d+$/] $port       = '80',
```

Comment:
The variable `$port` is used as a string - interpolated in another string in a `ensure_resource` [declaration](https://github.com/puppetlabs/puppetlabs-apache/pull/1679/files#diff-adb49d5dc0194dcc9856f3a221521c23R76)